bot.js

```diff
@@ -251,3 +251,35 @@ controller.hears(
   }
 );
 
+
+/*eslint-disable no-process-exit */
+
+function exitHandler(options, err) {
+
+  var messages = [
+    "Kdo zakopnul o ten kabe... ```obed-o-bot has left the room```"
+  ];
+  // pro všechny kanály kde jsme aktuálně přítomní
+  // bot.say(messages[0]);
+
+  if (options.cleanup) console.log('clean');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'console'.](https://www.codacy.com/app/ondra/obed-o-bot/file/2229873906/issues/source?bid=3165659&fileBranchId=3175486#l265)